Message formatting choices

mrvdb opened the pull request:
- `parse_mode=html` is a good choice for all sources so, that can be set in any case (at least with the usecases I know it's a good default) 
- `link_preview=True` is a bit more  complex and really should be configurable per source.

Source data:
- activity.object.`url` => back link to source activity, this should probably come first in the message 
- activity.object.`content` -> typically html content 
- activity.object.`attachment` -> array of media urls

There are 2 things to work out:

1. which of the source data elements need to be in the message
2. do we want `link_preview`, which takes the first link in the message content and uses some telegram magic to create a preview of the destination (not always possible)

Analysis for my usecases:

- `url`  should be the first in the message to link back to the source content.
- if `link_preview` is OFF we should add `content`, otherwise it will just be the link
- if `link_preview` is ON the content is most likely rendered for us automatically (and Telegram mostly does the right thing)

Ignoring `attachment` for now, this pleads for having `link_preview` set to ON for all incoming messages and just using the `url` as content after which Telegram takes care of the rest.

If attachments come into play we can:
1. ignore them and have the user visit the source
2. post a separate message per attachment (a link) with `link_preview` choices as above 
3. include them in the message (so we don't get a bombardment of messages, but just one)


Ideal situation would be a templated configuration per 'follow', but this may be too much.

For me personally, a configuration where the message content is just `url` with 'link_preview=True` and ignoring attachments would be just fine for me.

mrvdb committed:
Make arguments for message format explicit

- parse_mode=html is a good choice for all sources

link_preview is a bit more  complex and really should be configurable per source.

autogestion commented:
> a configuration where the message content is just url with 'link_preview=True` and ignoring attachments would be just fine for me.

It's fine for me too.
I've made simple test, by putting in tg chat url of AP post with attached image
![Screenshot_26](https://user-images.githubusercontent.com/1098257/57098111-057e7500-6d22-11e9-9b7a-2d3fa2922a86.png)
looks good)

mrvdb commented:
Perfect, I'll update the PR with just `url` and `parse_mode='html'` and `link_preview=True`

mrvdb committed:
URL of original activity is sufficient

- telegram takes care of content/images with link_preview; this
  works in most of the cases and is a simple solution

autogestion commented:
@mrvdb could you also increment `__version__` in  https://github.com/autogestion/pubgate-telegram/blob/master/pg_telegram/__init__.py#L1

mrvdb committed:
Increment version

mrvdb commented:
:+1: Thanks!
